src/screens/courses/cardTemplates/TextMediaCard/index.tsx

```diff
@@ -20,13 +20,16 @@ interface TextMediaCardProps {
   card: TextMediaType,
   index: number,
   isLoading: boolean,
+  setIsSwipeEnabled: (boolean) => void,
 }
 
-const TextMediaCard = ({ card, index, isLoading }: TextMediaCardProps) => {
+const TextMediaCard = ({ card, index, isLoading, setIsSwipeEnabled }: TextMediaCardProps) => {
   const [mediaHeight, setMediaHeight] = useState<number>(CARD_MEDIA_MAX_HEIGHT);
   const [mediaType, setMediaType] = useState<string>('');
   const [mediaSource, setMediaSource] = useState<{ uri: string } | undefined>();
 
+  useEffect(() => setIsSwipeEnabled(true));
```

Comment:
Est-ce que mettre `useEffect(() => setIsSwipeEnabled(true), []);` pourrait suffire ? Ou est-ce que quand on revient sur la carte, le useEffect ne sera pas triggered ?

Reply:
Non malheureusement. Je ne crois pas que le composant est unmounted lorsqu'on change de carte. Et `useEffect(() => setIsSwipeEnabled(true), []);` ne s'active qu'au mounted il me semble.
(https://medium.com/javascript-in-plain-english/lifecycle-methods-substitute-with-react-hooks-b173073052a)